docker/junit5/src/main/java/org/arquillian/cube/docker/junit5/NetworkDsl.java

```diff
@@ -0,0 +1,28 @@
+package org.arquillian.cube.docker.junit5;
+
+import org.arquillian.cube.docker.impl.client.containerobject.dsl.Network;
+import org.arquillian.cube.docker.impl.client.containerobject.dsl.NetworkBuilder;
+
+public class NetworkDsl {
```

Comment:
How about writing tests for `NetworkDsl`?

Reply:
Same

Reply:
this is fine for `NetworkDsl`